src/binary-reader-linker.cc

```diff
@@ -243,11 +261,16 @@ Result BinaryReaderLinker::OnExport(Index index,
                                     ExternalKind kind,
                                     Index item_index,
                                     StringSlice name) {
+  if (kind == ExternalKind::Memory) {
+    WABT_FATAL("Linker does not support exported memories");
+  }
+
```

Comment:
@sbc100 oh don't worry, I completely respect that not every tool will solve every use case I have. 👍  It's one of those "it would save me the trouble" requests.

Truth be told, I don't _really_ know what I'm doing. I'm use wasm as a chance to dig even deeper into my arm chair compiler knowledge and learning as I go. e.g. I don't yet know what the relocation section conventions even are for wasm and only today discovered [Linking.md](https://github.com/WebAssembly/tool-conventions/blob/master/Linking.md).

Right now my binaries are produced from my own compiler and backend directly and do not contain a relocation section. Eventually I'll likely switch to LLVM, but it was early when I started and was a good excuse to really learn the spec.

The likely correct solution for me to is to follow [Linking.md](https://github.com/WebAssembly/tool-conventions/blob/master/Linking.md) and actually create relocable binaries. Agreed?